extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -35,20 +34,25 @@ public function index() {
 		wfProfileIn( __METHOD__ );
```

Comment:
`wfProfileIn()` is deprecated. We already use `xhprof` for profiling so you can simply skip them.